src/python/pants/backend/python/dependency_inference/scripts/BUILD

```diff
@@ -0,0 +1,4 @@
+resources(name="resources", sources=["*.py"])
+
+# Also expose scripts as `python_sources` so they get formatted/linted/checked
+python_sources()
```

Comment:
It's really important that pyupgrade does not run on this file, as python 2.7 needs to run on it. We want to add skip_pyupgrade=True. But I don't think we activate the pyupgrade backend by default...so we'll want to add that backend to pants.toml but then set [pyupgrade].skip = true. Also update the [cli].alias appropriately

Reply:
That seems best left to a separate PR, I'll add a commented-out `skip_pyupgrade` for this one, and then when that one is made, we can uncomment-out.